package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bildvitta/vuex-offline",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Se está como Não publicado, não mude a versão no readme
```suggestion
  "version": "3.0.0",
```

---

Feito

---

Feito